test/jest/__mock__/stripesComponents.mock.js

```diff
@@ -2,14 +2,29 @@ import React from 'react';
 
 jest.mock('@folio/stripes/components', () => ({
   ...jest.requireActual('@folio/stripes/components'),
-  Accordion: jest.fn(({ children, label }) => (
+  Accordion: jest.fn(({ children, label, name, onClearFilter }) => (
     <div>
       {label}
       {children}
+      <div>
+        <button
+          type="button"
+          onClick={() => onClearFilter()}
+          data-testid={`clear-${name}`}
+        >Clear
+        </button>
+      </div>
+      <div data-testid={`accordion-${name}`} />
     </div>
   )),
-  AccordionSet: jest.fn(({ children }) => (
+  AccordionSet: jest.fn(({ children, onToggle }) => (
```

Comment:
I do not see the usage of this component inside `PickupServicePointFilter`. Do we really need to mock it?